.github/workflows/js-tests.yml

```diff
@@ -0,0 +1,41 @@
+name: Javascript Tests
+
+on: [push, pull_request]
```

Comment:
```suggestion
on:
  push:
    branches: ['develop']
  pull_request:
    branches: ['develop']
```
To remove duplicated workflows. See: <img width="1174" alt="Screenshot 2022-08-17 at 23 05 42" src="https://user-images.githubusercontent.com/78743291/185242976-c9f6eb2a-f997-4b91-999c-e2065f5b933f.png">

Reply:
Agreed, duplicate workflows needs to be resolved and though I may be wrong, I think doing `push` & `pull_request` only on `develop` branch wouldn't mirror our current process of triggering checks on every commit across branches in this repo. Going to look into some options.